### PR TITLE
[IMP] l10n_hu: Update available banks list

### PR DESCRIPTION
**Pull request by lordkrandel**
Lots of contact information has been changed and some banks has been sold their bisiness portfolio over the years. In some cases the telephone and email links were in the wrong order.

Some of the banks were merged or under Sanctions. (Sberbank, IC, MKB, Budapest, Commerz, FHB... )

Original credits to `kerteszgergo`: odoo/odoo#108301

### CODE REVIEW

File: addons/l10n_hu/data/res.bank.csv [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-"GIBAHUHB","ERSTE Bank Hungary Zrt.","GIBAHUHB","base.hu",1138,"Budapest","Népfürdő u. 24-26.","erste@erstebank.hu","+3640222221",
-"FHKBHUHB","FHB Kereskedelmi Bank Zrt.","FHKBHUHB","base.hu",1082,"Budapest","Üllői út 48."," info@fhb.hu","+3614529100"
-"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+3640100777"
-"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi  út 1-3."," level@bancopopolare.hu ","+3640200515",
-"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György  út 84.","communications.hu@ingbank.com","+362680140"
-"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9."," bank@kh.hu ","+3613289000"
-"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy utca 98.","info@magnetbank.hu","+3614288888"
+"GIBAHUHB","ERSTE Bank Hungary Zrt.","GIBAHUHB","base.hu",1138,"Budapest","Népfürdő u. 24-26.","erste@erstebank.hu","+3640222221"
+"FHKBHUHB","FHB Kereskedelmi Bank Zrt.","FHKBHUHB","base.hu",1082,"Budapest","Üllői út 48.","info@fhb.hu","+3614529100"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:10]: To be removed
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy út 98.","info@magnetbank.hu","+3614288888"
 "MANEHUHB","Magyar Nemzeti Bank","MANEHUHB","base.hu",1054,"Budapest","Szabadság tér 8-9.","info@mnb.hu","+3614282600"
 "MKKBHUHB","MKB Bank Zrt.","MKKBHUHB","base.hu",1056,"Budapest","Váci utca 38.","telebankar@mkb.hu","+3613278600"
-"OTPVHUHB","OTP Bank Nyrt.","OTPVHUHB","base.hu",1051,"Budapest","Nádor utca 16.",,"+3614735000"
-"UBRTHUHB","RAIFFEISEN Bank Zrt.","UBRTHUHB","base.hu",1054,"Budapest","Akadémia utca 6.","+3640484848",
-"MAVOHUHB","Sberbank Magyarország Zrt.","MAVOHUHB","base.hu",1088,"Budapest","Rákóczi  út 7.","info@sberbank.hu","+3614114200"
-"TAKBHUHB","TakarékBank Zrt.","TAKBHUHB","base.hu",1122,"Budapest","Pethényi  köz 10.","info@tbank.hu"
+"OTPVHUHB","OTP Bank Nyrt.","OTPVHUHB","base.hu",1051,"Budapest","Nádor utca 16.","otpbank@otpbank.hu","+3614735000"
+"UBRTHUHB","RAIFFEISEN Bank Zrt.","UBRTHUHB","base.hu",1054,"Budapest","Akadémia utca 6.","info@raiffeisen.hu","+3680488588"
+"TAKBHUHB","TakarékBank Zrt.","TAKBHUHB","base.hu",1117,"Budapest","Magyar tudósok körútja 9.","kozpont@takarek.hu","+3613113110"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:20]: To be removed
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
+"GIBAHUHB","ERSTE Bank Hungary Zrt.","GIBAHUHB","base.hu",1138,"Budapest","Népfürdő u. 24-26.","erste@erstebank.hu","+3640222221"
+"FHKBHUHB","FHB Kereskedelmi Bank Zrt.","FHKBHUHB","base.hu",1082,"Budapest","Üllői út 48.","info@fhb.hu","+3614529100"
+"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+3615100527"
+"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi út 1-3.","level@bancopopolare.hu","+3640200515"
+"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György út 84/b.","communications.hu@ingbank.com","+3612358700"
+"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9.","bank@kh.hu","+3613289000"
+"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy út 98.","info@magnetbank.hu","+3614288888"
 "MANEHUHB","Magyar Nemzeti Bank","MANEHUHB","base.hu",1054,"Budapest","Szabadság tér 8-9.","info@mnb.hu","+3614282600"
 "MKKBHUHB","MKB Bank Zrt.","MKKBHUHB","base.hu",1056,"Budapest","Váci utca 38.","telebankar@mkb.hu","+3613278600"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:17]: To be changed to MBH Bank
https://hu.wikipedia.org/wiki/MBH_Bank 
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
-"OTPVHUHB","OTP Bank Nyrt.","OTPVHUHB","base.hu",1051,"Budapest","Nádor utca 16.",,"+3614735000"
-"UBRTHUHB","RAIFFEISEN Bank Zrt.","UBRTHUHB","base.hu",1054,"Budapest","Akadémia utca 6.","+3640484848",
-"MAVOHUHB","Sberbank Magyarország Zrt.","MAVOHUHB","base.hu",1088,"Budapest","Rákóczi  út 7.","info@sberbank.hu","+3614114200"
-"TAKBHUHB","TakarékBank Zrt.","TAKBHUHB","base.hu",1122,"Budapest","Pethényi  köz 10.","info@tbank.hu"
+"OTPVHUHB","OTP Bank Nyrt.","OTPVHUHB","base.hu",1051,"Budapest","Nádor utca 16.","otpbank@otpbank.hu","+3614735000"
+"UBRTHUHB","RAIFFEISEN Bank Zrt.","UBRTHUHB","base.hu",1054,"Budapest","Akadémia utca 6.","info@raiffeisen.hu","+3680488588"
+"TAKBHUHB","TakarékBank Zrt.","TAKBHUHB","base.hu",1117,"Budapest","Magyar tudósok körútja 9.","kozpont@takarek.hu","+3613113110"
 "BACXHUHB","UniCredit Bank Hungary Zrt.","BACXHUHB","base.hu",1054,"Budapest","Szabadság tér 5-6.","info@unicreditgroup.hu","+3613011271"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:21]: Phone be changed to +3613253200
immagine
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 5]
```diff
@@ -1,22 +1,21 @@
 "id","name","bic","country/id","zip","city","street","email","phone"
 "BKCHHUHBXXX","Bank of China (Hungária) Hitelintézet Rt.","BKCHHUHBXXX","base.hu",1051,"Budapest","József Nádor tér 7.","service_hu@bank-of-china.com","+3614299200"
-"BNPAHUHX","BNP Paribas Hungária Bank Rt.","BNPAHUHX","base.hu",1051,"Budapest","Széchenyi István  tér 7-8.",
-"BUDAHUHB","Budapest Hitel- és Fejlesztési Bank Rt.","BUDAHUHB","base.hu",1138,"Budapest","Váci út 188."," info@budapestbank.hu","+3614506060"
+"BNPAHUHX","BNP Paribas Hungária Bank Rt.","BNPAHUHX","base.hu",1051,"Budapest","Széchenyi István  tér 7-8."
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:3]: Phone: +3613746300
Email: csd_hungary@bnpparibas.com
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 6]
```diff
@@ -1,22 +1,21 @@
...
+"BUDAHUHB","Budapest Hitel- és Fejlesztési Bank Rt.","BUDAHUHB","base.hu",1138,"Budapest","Váci út 188.","info@budapestbank.hu","+3614506060"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:4]: Defunct 2022, fused into MKB Bank, then fused into MBH Bank
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 7]
```diff
@@ -1,22 +1,21 @@
...
 "CIBHHUHB","CIB Közép-Európai Nemzetközi Bank Zrt.","CIBHHUHB","base.hu",1027,"Budapest","Medve utca 4-14.","cib@cib.hu","+3614231000"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:5]: Phone: +3614242242

immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 8]
```diff
@@ -1,22 +1,21 @@
...
 "CITIHUHX","Citibank Rt.","CITIHUHX","base.hu",1051,"Budapest","Szabadság tér 7.",,"+3613745000"
-"COBAHUHX","Commerzbank Zártkörűen Működő Rt.","COBAHUHX","base.hu",1054,"Budapest","Széchenyi  rakpart 8.","info.budapest@commerzbank.com","+3613748100"
+"COBAHUHX","Commerzbank Zártkörűen Működő Rt.","COBAHUHX","base.hu",1054,"Budapest","Széchenyi rakpart 8.","info.budapest@commerzbank.com","+3613748100"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:7]: Section bought by Erste Bank
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 9]
```diff
@@ -1,22 +1,21 @@
...
 "DEUTHU2B","Deutsche Bank Zártkörűen Működő Rt.","DEUTHU2B","base.hu",1054,"Budapest","Hold utca 27.","db.hungary@db.com","+3613013700"
-"GIBAHUHB","ERSTE Bank Hungary Zrt.","GIBAHUHB","base.hu",1138,"Budapest","Népfürdő u. 24-26.","erste@erstebank.hu","+3640222221",
-"FHKBHUHB","FHB Kereskedelmi Bank Zrt.","FHKBHUHB","base.hu",1082,"Budapest","Üllői út 48."," info@fhb.hu","+3614529100"
-"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+3640100777"
-"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi  út 1-3."," level@bancopopolare.hu ","+3640200515",
-"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György  út 84.","communications.hu@ingbank.com","+362680140"
-"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9."," bank@kh.hu ","+3613289000"
-"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy utca 98.","info@magnetbank.hu","+3614288888"
+"GIBAHUHB","ERSTE Bank Hungary Zrt.","GIBAHUHB","base.hu",1138,"Budapest","Népfürdő u. 24-26.","erste@erstebank.hu","+3640222221"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:9]: Phone +3612980222
immagine

File: addons/l10n_hu/data/res.bank.csv (hunk 1 above)
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:10]: To be removed, merged with Takarek, which is also merged...
immagine

File: addons/l10n_hu/data/res.bank.csv [hunk 10]
```diff
@@ -1,22 +1,21 @@
...
+"FHKBHUHB","FHB Kereskedelmi Bank Zrt.","FHKBHUHB","base.hu",1082,"Budapest","Üllői út 48.","info@fhb.hu","+3614529100"
+"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+3615100527"
+"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi út 1-3.","level@bancopopolare.hu","+3640200515"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:12]: immagine
IC bank is under sanctions.

BancoPopolare (see the website indicated here) has been sold to  MagNetBank

File: addons/l10n_hu/data/res.bank.csv [hunk 11]
```diff
@@ -1,22 +1,21 @@
...
+"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György út 84/b.","communications.hu@ingbank.com","+3612358700"
+"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9.","bank@kh.hu","+3613289000"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:14]: Phone is +36203353355
immagine

**Comment by lordkrandel** [comment]
Verified all existing banks with available public information.

**Review by aboo-odoo — COMMENTED** [review]
Nice job! LGTM @clbr-odoo

**Comment by clbr-odoo** [comment]
@lordkrandel Changes looks good thanks, although I'm wondering if it is a good idea to remove data record in stable.
With your change, if the user had a "to be removed" bank on some records, they'll end up with an empty field (since the occurrences I found are not `required`, it doesn't crash. But I guess it could ? :thinking: ).
I'm wondering if it wouldn't be safer to update data in stable, and only remove in master ?
We should probably also handle the merge of banks properly in an upgrade script ? :grimacing:

**Comment by lordkrandel** [comment]
You're totally right, what about deactivating them here?

**Comment by clbr-odoo** [comment]
Sounds good to me :+1:

**Review by clbr-odoo — APPROVED** [review]
Perfect thanks. :pray: 
For the record Paolo set the irrelevant banks to active=False in stable, and will do the real-cleanup in master.
@robodoo r+

**Pull request closed without merging**